mk/rabbitmq-tools.mk

```diff
@@ -390,3 +390,32 @@ clean-ct-logs-archive::
 	$(gen_verbose) rm -f $(PROJECT)-ct-logs-*.tar.xz
 
 clean:: clean-ct-logs-archive
+
+# --------------------------------------------------------------------
+# Generate a file listing RabbitMQ component dependencies and their
+# Git commit hash.
+# --------------------------------------------------------------------
+
+rmq-deps.mk: $(PROJECT)-rmq-deps.mk
```

Comment:
Can we call this `deps.mk` instead? If not, then my second proposal is `rabbitmq-server-deps.mk`.

Reply:
`server-deps.mk` should also be descriptive enough?

Reply:
The file lists the RabbitMQ component dependencies only, not all dependencies, thus the `rmq` in the name.

The file is named `$(PROJECT)-rmq.deps.mk`, not `rmq-deps.mk`. The latter is just a convenient shortcut to call the recipe. `server` does not make sense in this recipe because it will be executed for rabbitmq-common, the CLI and plugins in the future.

Reply:
In this case `$(PROJECT).deps.mk` feels appropriate.

Reply:
Ok, then `$(PROJECT).rabbitmq.deps.mk` makes most sense.

Reply:
Do you mind if we use dashes (`$(PROJECT)-rabbitmq-deps.mk`)? We never used dots so far in the middle of the files we generate.